core/src/main/java/hudson/cli/CreateJobHelper.java

```diff
@@ -0,0 +1,34 @@
+package hudson.cli;
+
+import hudson.model.Item;
+import hudson.model.TopLevelItem;
+import jenkins.model.Jenkins;
+import jenkins.model.ModifiableTopLevelItemGroup;
+
+public class CreateJobHelper {
```

Comment:
As you put the static method as "package protected", do you think it should be the same for the class?

From code external to the package, the class is an empty shell.

Reply:
Sorry, can you provide a suggested change?

I don't really have an opinion on any of these changes ...

Reply:
#4078